### PR TITLE
21600 - store for si control.

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:21600 *https://github.com/bcgov/entity/issues/21600 

*Description of changes:*
- converted SI control to be contained in the store

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-175-0tspnxv2.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-175-0tspnxv2.web.app

File: btr-web/btr-main-app/utils/btr-bods/bods-to-si-schema-converters.ts [hunk 1]
```diff
@@ -212,14 +213,14 @@ const _getSi = (
...
-    votesInConcert: getConnectedIndividuals(oocs, ControlOfVotesDetailsE.IN_CONCERT_CONTROL),
-    votesActingJointly: getConnectedIndividuals(oocs, ControlOfVotesDetailsE.ACTING_JOINTLY),
-    directorsInConcert: getConnectedIndividuals(oocs, ControlOfDirectorsDetailsE.IN_CONCERT_CONTROL),
-    directorsActingJointly: getConnectedIndividuals(oocs, ControlOfDirectorsDetailsE.ACTING_JOINTLY)
+    // sharesInConcert: getConnectedIndividuals(oocs, ControlOfSharesDetailsE.IN_CONCERT_CONTROL),
+    // sharesActingJointly: getConnectedIndividuals(oocs, ControlOfSharesDetailsE.ACTING_JOINTLY),
+    // votesInConcert: getConnectedIndividuals(oocs, ControlOfVotesDetailsE.IN_CONCERT_CONTROL),
+    // votesActingJointly: getConnectedIndividuals(oocs, ControlOfVotesDetailsE.ACTING_JOINTLY),
+    // directorsInConcert: getConnectedIndividuals(oocs, ControlOfDirectorsDetailsE.IN_CONCERT_CONTROL),
+    // directorsActingJointly: getConnectedIndividuals(oocs, ControlOfDirectorsDetailsE.ACTING_JOINTLY)
```
Comment (kialj876) [inline btr-web/btr-main-app/utils/btr-bods/bods-to-si-schema-converters.ts:223]: I think you can remove these comments?

File: btr-web/btr-main-app/utils/btr-bods/bods-to-si-schema-converters.ts [hunk 2]
```diff
@@ -234,3 +235,32 @@ export const getSIsFromBtrBodsSubmission = (submission: BtrFilingI): SiSchemaTyp
...
 }
+
+export const getSiControlConnectionsFromBodsSubmission =
+  (submission: BtrFilingI): Map<string, JointlyOrInConcertConnectionsI> => {
+    // sharesInConcert:
+    //   sharesActingJointly:
+    //   votesInConcert
+    //   votesActingJointly:
+    //   directorsInConcert
+    //   directorsActingJointly
```
Comment (kialj876) [inline btr-web/btr-main-app/utils/btr-bods/bods-to-si-schema-converters.ts:246]: and here too

File: btr-web/btr-main-app/tests/services/expected-ouptuts.ts [hunk 3]
```diff
@@ -34,7 +34,7 @@ export const OwnershipOrControlStatements = {
             minimum: 50
           },
           type: 'shareholding',
-          connectedIndividuals: [{ uuid: '002', legalName: 'Another test name' }]
+          connectedIndividuals: []
```
Comment (kialj876) [inline btr-web/btr-main-app/tests/services/expected-ouptuts.ts:37]: Do we have a test case that makes sure the connectedIndividual is in the submission ?

**Review by kialj876 — APPROVED** [review]
Nice work!

File: btr-web/btr-main-app/tests/services/expected-ouptuts.ts (hunk 3 above)
Comment (hfekete) [reply btr-web/btr-main-app/tests/services/expected-ouptuts.ts]: Added, thanks, will merge after all checks pass.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-175-0tspnxv2.web.app

**Pull request merged**